pandas/core/arrays/_mixins.py

```diff
@@ -189,6 +193,22 @@ def equals(self, other) -> bool:
     def _values_for_argsort(self):
         return self._ndarray
 
+    # Signature of "argmin" incompatible with supertype "ExtensionArray"
+    def argmin(self, axis: int = 0, skipna: bool = True):  # type:ignore[override]
+        # override base class by adding axis keyword
+        validate_bool_kwarg(skipna, "skipna")
+        if not skipna and self.isna().any():
+            raise NotImplementedError
+        return nargminmax(self, "argmin", axis=axis)
```

Comment:
-> nanargmin (is this not exercised in tests?)

Reply:
this is exercised in tests

The only difference between this and EA.argmin is this passes `axis`